packages/graph/src/graph2/legend.ts

```diff
@@ -0,0 +1,83 @@
+import { Vertex2 } from "@hpcc-js/react";
+import { Graph2 } from "./graph";
+
+export class GraphLegend extends Graph2 {
+
+    constructor() {
+        super();
+        this._iconBar.buttons([]);
+        this.vertexRenderer(Vertex2)
+            .layout("Hierarchy")
+            .hierarchyRankDirection("LR")
+            .hierarchyNodeSeparation(30)
+            .allowDragging(false)
+            .highlightOnMouseOverVertex(false)
+            ;
+    }
+
+    update(domNode, element) {
+
+        const dataHasChanged = true;
```

Comment:
@GordonSmith Is this check worth implementing? It seems like the only time we'd call render on GraphLegend is when the data changes anyway.

Reply:
Agree - and the data is "small"